src/components/Auction/index.jsx

```diff
@@ -1,5 +1,85 @@
 import React from 'react';
+import './style.css';
+import Corvette from '/src/components/Auction/auto.jpeg';
+import Winner from '/src/components/Auction/winner.png';
+import { BsShare } from 'react-icons/bs';
+import { ImEye } from 'react-icons/im';
+import { GiPayMoney } from 'react-icons/gi';
 
-export const Auction = () => {
-  return <h2>Aukce</h2>;
-};
+export const Auction = () => (
+  <div className="auction__container">
+    <div className="auction__site-share">
+      <h2 className="auction__site-title">Aukce XY</h2>
+
+      <button className="auction__share-btn">
+        {' '}
+        <BsShare /> {`Sdílet`.toUpperCase()}
+      </button>
+    </div>
+    <p className="auction__timekeeper">
+      0 dní 10 hodin 15 minut 43 sekund do konce
+    </p>
+    <img className="auction__pic-item1" alt="Děd s corvettou" src={Corvette} />
```

Comment:
Buď to vyřešit nějak tím layoutem, nebo nastavit obrázku max-width, aby se nezvětšoval přes celou obrazovku